src/functionalTest/resources/scenarios/RIA-1022-out-of-time-appeal-submission-document.json

```diff
@@ -0,0 +1,50 @@
+{
+  "description": "RIA-1022 Create out of time appeal submission PDF",
+  "request": {
+    "uri": "/asylum/ccdAboutToSubmit",
+    "credentials": "LegalRepresentative",
+    "input": {
+      "eventId": "submitAppeal",
+      "state": "appealSubmitted",
+      "caseData": {
+        "template": "minimal-appeal-submitted.json",
+        "replacements": {
+          "homeOfficeDecisionDate": "{$TODAY-30}",
+          "submissionOutOfTime": "Yes",
+          "applicationOutOfTimeExplanation": "outOfTimeExplanation",
+          "applicationOutOfTimeDocument": {
+            "document_url": "{$FIXTURE_DOC3_PDF_URL}",
```

Comment:
Not sure if you should be passing in the out of time document here.  I guess we are generating the document as part of the event callback so you'd expect one to come back but have one sent by ccd.

Reply:
If I understood correctly when I debug ia-case-documents-api I have seen out of time document attached to the case object. I need filename of the document to display in the template. Generated summary document in the event is a different one. 
It is difficult to validate that filename of provided out of date document is displayed in the summary pdf document generated in the event.

Reply:
Is that the document that the user attaches rather than the one that gets generated then?

Reply:
yes, this is document attached by user